keyboards/tweetydabird/lotus58/keymaps/default/rules.mk

```diff
@@ -1,2 +1,6 @@
 ENCODER_MAP_ENABLE = yes
 AUTO_SHIFT_ENABLE = yes
+
+ifeq ($(strip $(MCU)), atmega32u4)
+	LTO_ENABLE = yes
+endif
```

Comment:
Nah, just throw this into the info.json for the elite_c/promicro/nanoboot boards.  At this point, no reason not to, since avr will be space constrained on this board.

Reply:
Will do